source/reference/mongosync.txt

```diff
@@ -1,9 +1,95 @@
 .. _c2c-mongosync:
 
+.. default-domain:: mongodb
+
+.. program:: mongosync
+
 =============
 ``mongosync``
 =============
 
-.. default-domain:: mongodb
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Synopsis
+--------
+
+The ``mongosync`` binary is the primary process used in Cluster to Cluster Sync.
+It migrates data from one cluster to another and can keep the clusters
+in continuous sync.
+
+This document provides a complete overview of all command-line options.
+
+Options
+-------
+
+Global Options
+~~~~~~~~~~~~~~
+
+.. option:: --cluster0 <URI>
+
+   Sets the URI of the first cluster.  The first cluster can serve as either the
+   source cluster or the destination cluster in the synchronization process.
+
+
+.. option:: --cluster1 <URI>
+
+   Sets the URI of the second cluster.  The second cluster can serve as either
+   the source cluster or the destination cluster in the synchronization process.
+
+
+.. option:: --verbosity <level>
+
+   *Default*: ``INFO``
+
+   Sets the verbosity level to use in log messages.  Cluster to Cluster Sync logs
+   all messages at the specified level as well as any messages at less severe levels.
+
+   The ``--verbosity`` option supports the following values:
+
+   - ``TRACE``
+   - ``DEBUG``
+   - ``INFO``
+   - ``WARN``
+   - ``ERROR``
+   - ``FATAL``
+   - ``PANIC``
+
+
+.. option:: --logPath <DIR>
+
+  Sets the path to the log directory.   Cluster to Cluster Sync writes logs to
+  files in this directory.
+
+
+.. option:: --port
+
+   *Default*: ``27182``
+
+   Sets the port used by the HTTP server for the Cluster to Cluster Sync REST API.
+
+
+.. option:: --id <ID>
+
+   Sets the shard ID for the ``mongosync`` instance.
+
+   Use this option when running multiple instances of ``mongosync`` on the
+   same host.
+
+
+.. option:: --config <filename>
+
+   Sets the path to the configuration file.
+
+
+.. option:: --version, -v
+
+   Prints ``mongosync`` version information to stdout.
+
+
+.. option:: --help, h
 
-Migrate data between clusters using ``mongosync``.
+   Prints usage information to stdout.
```

Comment:
As mentioned above, consider adding Behavior and Example sections here.

Reply:
Fixed